backend/typescript/services/implementations/campService.ts

```diff
@@ -54,7 +96,7 @@ class CampService implements ICampService {
             endTime: campSession.endTime,
             registrations: campSession.campers.length,
             waitlist: campSession.waitlist.length,
-            active: campSession.active,
+            status: campSession.status,
```

Comment:
I think we should return the `camp` and `id` field here as well since we'll need the id to edit campSessions in the registration form